i18n/ja/messages.ja.xlf

```diff
@@ -226,7 +226,7 @@
       </trans-unit>
       <trans-unit id="8433565378732329468" datatype="html">
         <source><x id="INTERPOLATION" equiv-text="&quot;\uFFFD0\uFFFD&quot;"/> <x id="INTERPOLATION_1" equiv-text="&quot;\uFFFD1\uFFFD&quot;"/> will be updated to reflect the desired replicas count.</source>
-        <target><x id="INTERPOLATION" equiv-text="&quot;\uFFFD0\uFFFD&quot;"/> <x id="INTERPOLATION_1" equiv-text="&quot;\uFFFD1\uFFFD&quot;"/> は、目標のレプリカ数になるように更新されます。</target>
+        <target><x id="INTERPOLATION" equiv-text="{{ data.displayName }}"/> <x id="INTERPOLATION_1" equiv-text="{{ data.objectMeta.name }}"/> は、目標のレプリカ数になるように更新されます。</target>
```

Comment:
I don't know how to review the update which changes `"&quot;\uFFFD0\uFFFD&quot;"` to `"{{ data.displayName }}"`.

@shu-mutou 
Any comments?

Reply:
I checked the former translation [like this one](https://github.com/kubernetes/dashboard/blob/d2f2023824acd6eba6d1a6376be9b4da25d6efff/i18n/ja/messages.ja.xlf#L164 )  and noticed that all of the `&quot;\uFFFD0\uFFFD&quot;` have been changed to the value displayed in the following path, [like this](https://github.com/kubernetes/dashboard/blob/d2f2023824acd6eba6d1a6376be9b4da25d6efff/src/app/frontend/common/dialogs/download/template.html#L22), therefore I modified all of them using the same method. Could you confirm if my understanding is correct or not? Many thanks!

Reply:
Thank you for the detailed explanation, It looks correct :smiley:

Reply:
I've checked similar updates in this PR and it's OK.

Reply:
Thank you so much for you confirmation!

Reply:
Content of equiv-text attribute does not seem to affect to view. These INTERPOLATIONs would be replaced correctly when the angular builds the dashboard.

Reply:
Well noted with many thanks!